meteor/reactivize.js

```diff
@@ -152,7 +152,11 @@ Template.sortable.rendered = function () {
 		// Insert the new element at the end of the list and move it where it was dropped.
 		// We could insert it at the beginning, but that would lead to negative orders.
 		var sortSpecifier = {}; sortSpecifier[orderField] = -1;
-		event.data.order = templateInstance.collection.findOne({}, { sort: sortSpecifier, limit: 1 }).order + 1;
+		if (templateInstance.collection.find().fetch().length === 0) {
```

Comment:
`templateInstance.collection.find().count() === 0` [should be more fast](http://docs.mongodb.org/manual/reference/method/cursor.count/):
> Append the count() method to a find() query to return the number of matching documents. The operation does not perform the query but instead counts the results that would be returned by the query.